cmd/ipfs/daemon.go

```diff
@@ -178,8 +178,8 @@ Headers.
 		cmds.BoolOption(enableGCKwd, "Enable automatic periodic repo garbage collection"),
 		cmds.BoolOption(adjustFDLimitKwd, "Check and raise file descriptor limits if needed").WithDefault(true),
 		cmds.BoolOption(migrateKwd, "If true, assume yes at the migrate prompt. If false, assume no."),
-		cmds.BoolOption(enablePubSubKwd, "Instantiate the ipfs daemon with the experimental pubsub feature enabled."),
-		cmds.BoolOption(enableIPNSPubSubKwd, "Enable IPNS record distribution through pubsub; enables pubsub."),
+		cmds.BoolOption(enablePubSubKwd, "Enable experimental pubsub feature. Overrides Pubsub.Enabled config and other pubsub flags."),
+		cmds.BoolOption(enableIPNSPubSubKwd, "Enable IPNS over pubsub. Implicitly enables pubsub, overrides Ipns.UsePubsub config."),
```

Comment:
@aschmahmann mind :eyes: ?

Reply:
> and other pubsub flags

seems a little vague, but perhaps is enough to indicate "setting to false will override Ipns.UsePubsub" without adding too much complexity to the help text.

Reply:
I guess it also implies that `--enable-namesys-pubsub=true --enable-pubsub-experiment=false` will fail which isn't true for the config file.

btw the behavior here is technically breaking, even if only for people doing weird things. Worth flagging anywhere?

Reply:
ya.... I think we made this worse. Let's just have the daemon flags map 1:1 to the config flags.